src/mito2/src/access_layer.rs

```diff
@@ -38,6 +38,9 @@ use crate::sst::parquet::reader::ParquetReaderBuilder;
 use crate::sst::parquet::writer::ParquetWriter;
 use crate::sst::parquet::{SstInfo, WriteOptions};
 
+/// Path prefix of the atomic write path.
+pub(crate) const ATOMIC_FS_PATH: &str = ".tmp/";
```

Comment:
Update this once https://github.com/GreptimeTeam/greptimedb/pull/6109 is merged.